smriprep/workflows/anatomical.py

```diff
@@ -222,12 +233,34 @@ def init_anat_preproc_wf(
                             run_without_submitting=True)
 
     # 2. Brain-extraction and INU (bias field) correction.
-    brain_extraction_wf = init_brain_extraction_wf(
-        in_template=skull_strip_template.space,
-        template_spec=skull_strip_template.spec,
-        atropos_use_random_seed=not skull_strip_fixed_seed,
-        omp_nthreads=omp_nthreads,
-        normalization_quality='precise' if not debug else 'testing')
+    def _is_skull_stripped(imgs):
+        """Check if T1w images are skull-stripped by interrogating
+        extreme planes for all (near) zeros.
+        """
+        def _check_img(img):
+            data = np.abs(nb.load(img).get_fdata(dtype=np.float32))
+            sidevals = data[0, :, :].sum() + data[-1, :, :].sum() + \
+                data[:, 0, :].sum() + data[:, -1, :].sum() + \
+                data[:, :, 0].sum() + data[:, :, -1].sum()
+            return sidevals < 10
+
+        return all(_check_img(img) for img in imgs)
```

Comment:
as we are delaying this definition, let's delay it to the latest possible (added suggestion below)
```suggestion
```